multilep/test/testing/tests.log

```diff
@@ -1,42 +1,57 @@
-commit 71cd5808f29530fb8ebe3aacb95d4808e83cc573
+commit 0df76f19c1e46cd98107ecd3484526bce106257d
 Author: tomc <tomc@cern.ch>
-Date:   Thu Mar 26 20:23:32 2020 +0100
+Date:   Thu Mar 26 21:03:20 2020 +0100
 
-    No need to have different versions for miniIsoCharged
-    
-    Effective areas have of course no effect on the charged component
+    Small fix
 
 --------------------------------------------------------------------------------------------------
 
 Running test: Run2018-17Sep2018 --> OK
-   Removed: _miniIsoCharged_80X,_miniIsoCharged_Spring15
+   Changed:
+      _miniIso_80X                                       mean:   0.2014 -->   0.1991     rms:   0.4842 -->   0.4833    
+      _relIso_80X                                        mean:   0.2742 -->   0.2681     rms:   0.5991 -->   0.5962    
```

Comment:
In the earlier commit, earlier in this pull request, I accidentally put again the 94X effective areas also  in these 80X variables. Fixed now.